robot/impl/local_robot.go

```diff
@@ -193,23 +193,23 @@ func (r *localRobot) StopAll(ctx context.Context, extra map[resource.Name]map[st
 	}
 
 	// Stop all stoppable resources
-	resourceErrs := []string{}
+	resourceErrs := make(map[string]error)
```

Comment:
@cheukt okay with this change? I think it's good for the StopAll error map in general.